_includes/head.html

```diff
@@ -11,7 +11,6 @@
     {% endcapture %}
     {{ styles | scssify }}
   </style>
-  <link rel="canonical" href="{{ page.url | replace:'index.html','' | absolute_url }}">
```

Comment:
this is already addressed by jekyll-seo-tags plugin and having two canonical metatags actually made these invalid